larrecodnn/ImagePatternAlgs/Tensorflow/Modules/waveformroifinder.fcl

```diff
@@ -3,8 +3,50 @@ BEGIN_PROLOG
 standard_waveformroifinder:
 {
     module_type: "WaveformRoiFinder"
+    NPlanes:         3
+    WaveformSize:    6000
+    RawProducerLabel:   ""
     WireProducerLabel:  "caldata:dataprep"
-    WaveformRecog: {
+
+    WaveformRecogPlane0: {
+       NNetModelFile: "CnnModels/lightmodel112.pb"
+       NNetOutputPattern: [
+          "cnn_output",
+          "dense_3"
+       ]
+       #TrtisModelName:     "lightmodel112"
+       #TrtisURL:          "localhost:8001"
+       #TrtisModelVersion:  -1
+       #TrtisVerbose:      true
+       WaveformSize:       6000
+       ScanWindowSize:     200
+       StrideLength:       150
+       MeanFilename:       "CnnModels/wvrec-mean.txt"
+       ScaleFilename:      "CnnModels/wvrec-scale.txt"
+       CnnPredCut:         0.5
+       tool_type: "WaveformRecogTf"
+    }
+
+    WaveformRecogPlane1: {
+       NNetModelFile: "CnnModels/lightmodel112.pb"
```

Comment:
How does the user get access to the "CnnModels/lightmodel112.pb" file?  Is it already included in a PR somewhere?  Or do they get it through CVMFS?

Reply:
@knoepfel Thank you so much for your detailed suggestions. Those are really helpful. 
Changes are made except the one related to "CnnModels/lightmodel112.pb". This part is along with the experiment release and can get it through CVMFS.   Regarding the tool for each plane, yes, we want to keep the individual tool for each plane, though the configuration here is using the same one, which can be easily replaced in the experiment fcl file.
Let me know if there are other  issues. Again, many thanks for your comments!